domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/MissingHeaderNotice.java

```diff
@@ -25,7 +25,7 @@
 
     public MissingHeaderNotice(final String filename, final String missingHeaderName) {
         super(filename, E_001,
-                "Field name not defined in the GTFS specification",
```

Comment:
Just flagging unrelated changes - not sure if they are intentional. A few others exist in this PR as well.

Reply:
Ok yes. This one is intentional  (the message is wrong)